backend/api/Models/UserModel.cs

```diff
@@ -20,11 +19,9 @@ public class UserModel : BaseModel, IEquatable<UserModel>
         public string LastName { get; set; }
 
         public string Email { get; set; }
+        #endregion
 
-        public IEnumerable<AgencyModel> Agencies { get; set; }
```

Comment:
I think the frontend is depending on these fields right now.

Reply:
I'll see if I can add them back in.

Reply:
Had to update automapper profiles to allow for them to come back.